pkg/qtum/rpc_types.go

```diff
@@ -88,12 +88,16 @@ type (
 		Headers    int64   `json:"headers"`
 		Mediantime int64   `json:"mediantime"`
 		Pruned     bool    `json:"pruned"`
-		Softforks  []struct {
-			ID     string `json:"id"`
-			Reject struct {
-				Status bool `json:"status"`
-			} `json:"reject"`
-			Version int64 `json:"version"`
+		Softforks  map[string]struct {
```

Comment:
this is where the main changes occur if you want to skip all the gofmts and the one truffle change